packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -1807,6 +1807,28 @@ export function lowercaseObjectKeys(
     )
 }
 
+export function validateRawBoolean<
```

Comment:
This is quite specific to what we do in `rawToEnriched`, so it seems more natural to put the function into that module. Feel free to keep it here, if you disagree.